tests/views/index.jade

```diff
@@ -1,12 +1,14 @@
 extends layout
 
 append scripts
+    script(src='../node_modules/babel-polyfill/dist/polyfill.js')

```

Comment:
I didn't run into a problem for Chrome or Firefox--I was adding it:

1. for parity with Karma/PhantomJS (which did need it)
2. any browsers that haven't yet supported it (which apparently may be IE/Edge per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility )

`Array.prototype.includes` is an ES2016 (ES7) feature, so a little bit later coming on...